Dockerfile

```diff
@@ -1,50 +1,113 @@
 FROM debian:jessie
 MAINTAINER ixkaito <ixkaito@gmail.com>
 
-RUN apt-get update \
-    && apt-get upgrade -y \
-    && apt-get clean \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-      apache2 \
-      libapache2-mod-php5 \
-      php5 \
-      php5-cli \
-      php5-gd \
-      php5-mysql \
-      php5-curl \
-      mysql-server \
-      mysql-client \
-      curl \
-      supervisor \
-      ca-certificates \
-      vim \
-      less \
-    && rm -rf /var/lib/apt/lists/*
+ENV TERM "xterm"
+
+ENV BUILDPCKGS "autoconf automake autopoint autotools-dev binutils bsdmainutils \
+	build-essential bzip2 cpp cpp-4.9 debhelper dh-php5 dpkg-dev fakeroot file \
+	g++ g++-4.9 gcc gcc-4.9 gettext gettext-base groff-base intltool-debian \
+	libalgorithm-diff-perl libalgorithm-diff-xs-perl libalgorithm-merge-perl \
+	libasan1 libasprintf-dev libasprintf0c2 libatomic1 libc-dev-bin libc6-dev \
+	libcilkrts5 libcloog-isl4 libcroco3 libdpkg-perl libfakeroot \
+	libfile-fcntllock-perl libgcc-4.9-dev libgettextpo-dev libgettextpo0 \
+	libglib2.0-0 libglib2.0-data libgomp1 libisl10 libitm1 liblsan0 \
+	libltdl-dev libltdl7 libmail-sendmail-perl libmpc3 libmpfr4 libpcre3-dev \
+	libpcrecpp0 libpipeline1 libquadmath0 libsigsegv2 libssl-dev libssl-doc \
+	libstdc++-4.9-dev libsys-hostname-long-perl libtimedate-perl libtool \
+	libtsan0 libubsan0 libunistring0 linux-libc-dev m4 make man-db manpages \
+	manpages-dev patch php-pear php5-dev pkg-php-tools po-debconf \
+	shared-mime-info shtool xdg-user-dirs xz-utils zlib1g-dev"
+
+RUN sed -xe \
+	DEBIAN_FRONTEND=noninteractive \
+	apt-get update \
+	apt-get upgrade -y \
+	apt-get clean \
+	apt-get install -y --no-install-recommends \
+		apache2 \
+		ca-certificates \
+		curl \
+		less \
+		libapache2-mod-php5 \
+		lsb-release \
+		mysql-server \
+		mysql-client \
+		nano \
+		php5 \
+		php5-cli \
+		php5-curl \
+		php5-dev \
+		php5-gd \
+		php5-mysql \
+		php5-xdebug \
+		supervisor \
+		vim \
+	&& touch /etc/php5/cli/conf.d/30-xdebug.ini \
+	&& echo " \
+		zend_extension=/usr/lib/php5/20131226/xdebug.so \
+		xdebug.overload_var_dump = 1 \
+		xdebug.var_disply_max_depth = -1 \
+		xdebug.var_display_max_children = -1 \
+		xdebug.var_display_max_data = 512 \
+		xdebug.max_nesting_level = -1 \
+		xdebug.collect_params = 4 \
+		xdebug.profiler_enable_trigger = 1 \
+		xdebug.profiler_enable = 0 \
+		xdebug.remote_enable = 1 \
+		xdebug.profiler_output_dir = \"/tmp\" \
+		xdebug.profiler_output_name = \"cachegrind.out.%t-%s\"" \
+			> /etc/php5/cli/conf.d/30-xdebug.ini \
+	&& apt-get remove ${BUILDPCKGS} \
```

Comment:
Some packages are not installed, so not removed, and this returned an error and stopped building.